### DISCUSSION TITLE
Nill Pill video questionnaire completion leads to MetaMask Phishing Detection warning.

### DISCUSSION THREAD

**Discussion by buidles** [NillionNetwork/discussions#159] [General]
After completing the questionnaire to the first Nill Pill video the user is redirected to https://metamask.github.io/phishing-warning/v4.1.0/#hostname=nillpill.com&href=https%3A%2F%2Fnillpill.com%2Fblindcomputation 

![Screenshot 2024-12-23 at 20 53 06](https://github.com/user-attachments/assets/8d24617d-4aa1-4806-966a-512cb6fd5768)

**Comment by crypblizz8** [comment]
Thanks for the flag. Looking into it
